istio-proxy/istio-proxy.spec

```diff
@@ -40,44 +39,52 @@ BuildRequires:  gcc
 BuildRequires:  gcc-c++
 BuildRequires:  make
 BuildRequires:  patch
-BuildRequires:  ksh
 BuildRequires:  xz
+BuildRequires:  git
 BuildRequires:  golang
 BuildRequires:  automake
 BuildRequires:  python3
 BuildRequires:  cmake3
 BuildRequires:  openssl
 BuildRequires:  openssl-devel
 BuildRequires:  libatomic
+BuildRequires:  platform-python-devel
 
-Source0:        istio-proxy.%{checksum}.tar.gz
-Source1:        build.sh
-Source2:        test.sh
```

Comment:
Can the test.sh file be removed?

Reply:
All files but the source .tar.gz were removed. We're invoking tests directly from .spec. Very straightforward as it should be.

Reply:
Nevermind. I see test.sh being removed now. Had to do a refresh to get it to show.